### DISCUSSION TITLE
Build server mock to trigger 'status_change_notification' of subscription handler on client

### DISCUSSION THREAD

**Discussion by stefan-swoboda** [FreeOpcUa/opcua-asyncio#1108] [Q&A]
Dear all,

I am currently working on a component that shall act as a OPC UA subscription client against a Siemens S7 PLC.  I need to do the
major part of the development without access to the PLC. During early integration tests I noticed that the PLC indeed issues
a subscription change notification under certain circumstances.

Now, I would like to find a way to build a sever mock that does invoke my status_change_notification method in my subscription handler. 

THX.

**Comment by schroeder-** [comment]
``` Python
sub_serv = server.iserver.subscription_service
status = ua.StatusCode(ua.StatusCodes.BadTimeout)
for sub in sub_serv.subscriptions.values():
    await sub.monitored_item_srv.trigger_statuschange(status)
```

**Reply by stefan-swoboda** [reply]
Thank you, works fine ! :-)
